From d5951b175d077ebefc67279c10c63c73171b9436 Mon Sep 17 00:00:00 2001
From: "yi.liu" <yi.liu@amlogic.com>
Date: Wed, 26 Apr 2023 17:37:19 +0800
Subject: [PATCH] amlbian: support debian minimal [1/1]

PD#SWPL-95583

Problem:
support debian minimal.

Solution:
support debian minimal.

Verify:
local

Signed-off-by: yi.liu <yi.liu@amlogic.com>
Change-Id: If259778798c124b98184981af4a1544a3cfc65da
---
 config-t7-bullseye-minimal.conf | 23 +++++++++++++++++++++++
 1 file changed, 23 insertions(+)
 create mode 100755 config-t7-bullseye-minimal.conf

diff --git a/config-t7-bullseye-minimal.conf b/config-t7-bullseye-minimal.conf
new file mode 100755
index 0000000..2715ccd
--- /dev/null
+++ b/config-t7-bullseye-minimal.conf
@@ -0,0 +1,23 @@
+#####################################
+#    Configuration File Template    #
+#####################################
+
+#WORK_BOARD - Supported Amlogic Board, check 'config/boards/amlogic/*.conf'
+#LINUX - Supported Linux version, check array 'SUPPORTED_LINUX' in 'config/boards/amlogic/*WORK_BOARD*.conf'
+#UBOOT - Supported U-boot version, check array 'SUPPORTED_UBOOT' in 'config/boards/amlogic/*WORK_BOARD*.conf'
+#DISTRIBUTION - Supported Distribution, check array 'DISTRIBUTION_ARRAY' in 'env/setenv.sh'
+#DISTRIB_RELEASE - Supported Distribution Release, check array '*DISTRIBUTION*_RELEASE_ARRAY' in 'env/setenv.sh'
+#DISTRIB_TYPE - Supported Distribution Type, check array '*DISTRIBUTION*_TYPE_ARRAY' in 'env/setenv.sh'
+#DISTRIB_ARCH - Supported Distribution Architecture, check array 'DISTRIB_ARCH_ARRAY' in 'env/setenv.sh'
+#INSTALL_TYPE - Supported Installation Type, check array 'INSTALL_TYPE_ARRAY' in 'env/setenv.sh'
+
+WORK_BOARD=T7-AN400
+LINUX=5.15
+UBOOT=2019.01
+DISTRIB_ARCH=arm64
+INSTALL_TYPE=EMMC
+
+DISTRIBUTION=Debian
+DISTRIB_RELEASE=bullseye
+DISTRIB_TYPE=minimal
+DOWNLOAD_MIRROR=default
-- 
2.25.1

